addons/api/mirage/factories/session.js

```diff
@@ -41,6 +41,29 @@ export default factory.extend({
         }
       }
     }
+  }),
+
+  /**
+   * Sessions generated with this trait will automatically form
+   * associations to target, host set, and host (if available) within
+   * the current scope and for specified user.
+   */
+  withUserAssociations: trait({
```

Comment:
This is essentially a duplicate of `withAssociations` on the sessions factory.  Why is this necessary?

Reply:
Removed.